.github/pull_request_template.md

```diff
@@ -36,3 +36,38 @@ _Please delete options that are not relevant._
 - [ ] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
 - [ ] My changes generate no new warnings
 - [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.
+
+## Project Metadata
```

Comment:
I was thinking of updating the PR template too, it contains a lot of unnecessary fields. You can remove these sections:
 
Make this line as comment:
![image](https://user-images.githubusercontent.com/43691873/117539794-d514f400-b029-11eb-8892-2b676b7ec475.png)

Remove this section:
![image](https://user-images.githubusercontent.com/43691873/117539810-e78f2d80-b029-11eb-999a-267ee7076df2.png)